### ISSUE TITLE
app: NextCloud

### ISSUE THREAD

**Issue by surferking** [atomery/webcatalog#751]
# App Submission

- **Name:** NextCloud
- **URL:** https://nextcloud.org
- **Category:** Productivity
- **Icon:** https://images.app.goo.gl/rU5d6XR5JiSp7cxZ8

![](https://user-images.githubusercontent.com/29705727/40315141-75a684e2-5d12-11e8-9732-b9bf680d8595.png)

**Comment by linonetwo** [comment]
Hi @surferking , can you help add those app to https://github.com/atomery/catalog ? 

So @quanglam2807  can have more time to code, implementing features :) , many thanks!

**Comment by surferking** [comment]
> Hi @surferking , can you help add those app to https://github.com/atomery/catalog ?
> 
> So @quanglam2807 can have more time to code, implementing features :) , many thanks!

Fair enough! OK, lemme see what I can do, otherwise I might recruit a colleague ;)
